src/Workspaces/Core/Portable/CodeStyle/CodeStyleOptions.cs

```diff
@@ -221,5 +221,68 @@ private static CodeStyleOption<AccessibilityModifiersRequired> ParseAccessibilit
 
             return s_requireAccessibilityModifiersDefault;
         }
+
+        private static readonly CodeStyleOption<ParenthesesPreference> s_defaultBinaryOperatorsParenthesesPreference =
+            new CodeStyleOption<ParenthesesPreference>(ParenthesesPreference.AlwaysForClarity, NotificationOption.None);
+
+        private static readonly CodeStyleOption<ParenthesesPreference> s_defaultOtherOperatorsParenthesesPreference =
+            new CodeStyleOption<ParenthesesPreference>(ParenthesesPreference.NeverIfUnnecessary, NotificationOption.None);
+
+        private const string s_otherOperatorsName = "other_operators";
+        private static PerLanguageOption<CodeStyleOption<ParenthesesPreference>> CreateParenthesesOption(
+            string fieldName, CodeStyleOption<ParenthesesPreference> defaultValue, string styleName)
+        {
+            var isOther = s_otherOperatorsName == styleName;
+            return new PerLanguageOption<CodeStyleOption<ParenthesesPreference>>(
+                nameof(CodeStyleOptions), fieldName, defaultValue,
+                storageLocations: new OptionStorageLocation[]{
+                    new EditorConfigStorageLocation<CodeStyleOption<ParenthesesPreference>>(
+                        $"dotnet_style_parentheses_in_{styleName}",
```

Comment:
❕ I frequently use text searches to find the definitions of .editorconfig options. Don't use formatting in a manner that prevents the complete option name from appearing as a literal in source code.

Reply:
sure.  this was more relevant when there were like 9 options. now that's 3, just repeating the full name is reasonably.

Reply:
fixed.